markup_tag_evaluation/evaluate_markup_tags.py

```diff
@@ -40,40 +40,45 @@ def main() -> None:
     args = parse_args()
     reference = read_text(args.reference)
     hypothesis = read_text(args.hypothesis)
+
+    backup_hypothesis = None
     if args.backup_hypothesis is not None:
         backup_hypothesis: Optional[List[str]] = read_text(args.backup_hypothesis)
```

Comment:
```suggestion
    backup_hypothesis: Optional[List[str]] = None
    if args.backup_hypothesis is not None:
        backup_hypothesis = read_text(args.backup_hypothesis)
```

Reply:
I think this is required for the type checks (if not we can skip the type annotation completely)